CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# Development flow
+
+
+## Building
+
+To build `flyctl`, all you need to do is to run `make build` from the root directory. This will build a binary in the `bin/` directory. Alternatively, you can run `go build .`
+
+To run `flyctl`, you can just run the binary you built using `make build`: `./bin/flyctl`. So for example, to update a machine, you can run `./bin/flyctl m update -a <app_name> <machine_id>`. Alternatively, you can build and run in the same command by running `go run .`, followed by whatever sub-command you want to run. Just note that this will have a slower startup.
+
+
+## Testing
+
+We have two different kinds of tests in `flyctl`, unit tests and integration tests (preflight). It's recommended to write a test for any features added or bug fixes, in order to prevent regressions in the future.
+
```

Comment:
Can you put the command to run tests in a code block?

Reply:
do you mean like in the below sections regarding tests, or more than that?

Reply:
Oh just a command to run. People will be skimming this thinking "okay what do I run?" -- just need to provide some copy/pasteable commands to get them started